src/converters/arguments.h

```diff
@@ -23,10 +23,25 @@
 
 namespace node_webrtc {
 
+struct Arguments {
+ public:
+  Nan::NAN_METHOD_ARGS_TYPE info;
+  explicit Arguments(Nan::NAN_METHOD_ARGS_TYPE info): info(info) {}
+};
```

Comment:
I hit some issues with ambiguous template specialization. Might be able to revert this change.